les/client.go

```diff
@@ -53,18 +54,20 @@ import (
 type LightEthereum struct {
 	lesCommons
 
-	peers              *serverPeerSet
-	reqDist            *requestDistributor
-	retriever          *retrieveManager
-	odr                *LesOdr
-	relay              *lesTxRelay
-	handler            *clientHandler
-	txPool             *light.TxPool
-	blockchain         *light.LightChain
-	serverPool         *vfc.ServerPool
-	serverPoolIterator enode.Iterator
-	pruner             *pruner
-	merger             *consensus.Merger
+	peers                *serverPeerSet
+	reqDist              *requestDistributor
+	retriever            *retrieveManager
+	odr                  *LesOdr
+	relay                *lesTxRelay
+	handler              *clientHandler
+	txPool               *light.TxPool
+	blockchain           *light.LightChain
+	serverPool           *vfc.ServerPool
+	serverPoolIterator   enode.Iterator
+	pruner               *pruner
+	merger               *consensus.Merger
+	seqRPCService        *rpc.Client
+	historicalRPCService *rpc.Client
```

Comment:
Nit: By adding a newline above the new fields we can keep the formatting independent (less noisy diff) and the optimism-addition more clear.

Reply:
Done, and agree that those big `go fmt` diffs are a bit noisy.